website/src/docs/companion.md

```diff
@@ -481,6 +486,25 @@ app.use(companion.app({
 
 We have [a detailed guide on running Companion in Kubernetes](https://github.com/transloadit/uppy/blob/main/packages/%40uppy/companion/KUBERNETES.md) for you, that’s how we run our example server at <https://companion.uppy.io>.
 
+### Running many instances
```

Comment:
Since we've been running Companion for many customers for a while, it would be great if we could add some data on when another instance is needed? Do we have any insight and how much a single instance can handle?

Reply:
Can you copy and edit this message: https://community.transloadit.com/t/performance-testing-of-uppy-companion/16157/3?u=merlijn

Reply:
I think in general for Node.js server apps it's always recommended to run more than one instance on each server, because if some code in one request blocks the event loop, all other requests get blocked too if only one instance. When uploading a lot of tiny files in parallel, the event loop could get saturated fast. I think the number of instances to run would usually be limited the amount of system memory as well as number of CPUs.